core/modules/main/templates/_reportissuecontainer.inc.php

```diff
@@ -4,6 +4,6 @@
         <a href="javascript:void(0);" class="closer" onclick="TBG.Main.Helpers.Backdrop.reset();"><?= fa_image_tag('times'); ?></a>
     </div>
     <div id="backdrop_detail_content" class="backdrop_detail_content">
-        <?php include_component('main/reportissue', compact('selected_project', 'issue', 'issuetypes', 'selected_issuetype', 'selected_milestone', 'selected_build', 'parent_issue', 'errors', 'permission_errors', 'board', 'locked_issuetype')); ?>
+        <?php include_component('main/reportissue', compact('selected_project', isset($issue)?'issue':array(), 'issuetypes', 'selected_issuetype', 'selected_milestone', 'selected_build', 'parent_issue', 'errors', isset($permission_errors)?'permission_errors':array(), 'board', 'locked_issuetype')); ?>
```

Comment:
See previous comments about initializing the methods in the associated component method